packages/tools/locales/ko.po

```diff
@@ -1,20 +1,22 @@
 # SOME DESCRIPTIVE TITLE.
 # Copyright (C) YEAR Laurent Cozic
 # This file is distributed under the same license as the Joplin-CLI package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
-#
+# FIRST Translator Ji-Hyeon Gim <potatogim@potatogim.net>, YEAR.
+# SENCOND Translator Hye-Rim Kim <kissofsea@gmail.com>, 2024.
 msgid ""
 msgstr ""
-"Project-Id-Version: Joplin-CLI 1.0.0\n"
+"Project-Id-Version: Joplin 2.14.16\n"
 "Report-Msgid-Bugs-To: \n"
-"Last-Translator: Ji-Hyeon Gim <potatogim@potatogim.net>\n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: Hye-Rim Kim <kissofsea@gmail.com>\n"
```

Comment:
Just out of curiosity. Aren't Korean names written with the surname first?
It's confusing, because sometimes they are not. It's easy to figure out in your case, but not so easy with names like `Bak Yoon` (actor), although I am pretty sure that `Yoon` is the surname.

Reply:
Yes, every korean name uses "Family name + Given name". :)
My name is "김혜림". My english name is "Kim Hye-Rim".

But many foreigners don't know that, so many korean use our last names in reverse.(Kim Hye-Rim -> Hye-Rim Kim)
For example, 김연아(figure skater) is "Kim Yuna". But she uses "Yuna Kim".

For reference, "Kim, Lee, Park, Jang, etc..." is Family name.
"Kim" means "Gimhae-Kim" or "Andong-Kim" or "Gwangun-Kim" or etc...

Reply:
Ah, thanks. 

> For example, 김연아(figure skater) is "Kim Yuna". But she uses "Yuna Kim".

If it's written in Korean it's easy to find out. But when it comes to the actor I mentioned earlier, on imdb there's only the not Korean version of his name. ;-)

> For reference, "Kim, Lee, Park, Jang, etc..." is Family name.
"Kim" means "Gimhae-Kim" or "Andong-Kim" or "Gwangun-Kim" or etc...

Ah, I see, the clan name. Which is not part of the official name I take it.

Reply:
Right, "Yoon" is Family name. Bak Yoon(actor) is "윤박" or "Yoon Bak" in Korea.
Sometimes given name is single word... :)
And his name is difficult, because "Yoon" and "Bak" use family name in Korea.
So Koreans are also confused... "Is his name Yoon? or bak?"...lol...

Reply:
Haha, good point. Thanks to you I now know that his first name is Bak (or acrtually Bag). It's interesting how the Korean consonants and vowls are transliterated into English: 윤박 should be Yun Bag (even though the B sounds more like a P and the G closer to a K).

Reply:
국어의 로마자 표기법 (Revised Romanization of Korean 또는 RRK)
It is a guideline of the Korean Language Institute in the "국립국어원".
The name name person can choose freely according to the person. XD
![Korean-English(2000)](https://github.com/laurent22/joplin/assets/151744644/75918960-9aa1-4fcb-aadb-710ddd6f36d6)
![Korean-English-name](https://github.com/laurent22/joplin/assets/151744644/daedaab0-eaca-4581-aaea-c49c431cf04c)